src/box/memtx_hash.c

```diff
@@ -567,7 +568,8 @@ static const struct index_vtab memtx_hash_index_vtab = {
 	/* .max = */ generic_index_max,
 	/* .random = */ memtx_hash_index_random,
 	/* .count = */ memtx_hash_index_count,
-	/* .get = */ memtx_hash_index_get,
+	/* .get_raw = */ memtx_hash_index_get_raw,
+	/* .get = */ memtx_index_get,
```

Comment:
Please see my comment to memtx tree. Note that you would have to convert c to cc.